pytest_fixtures/component/maintain.py

```diff
@@ -25,7 +25,7 @@ def module_stash(request):
 def sat_maintain(request, module_target_sat, module_capsule_configured):
     if settings.remotedb.server:
         sat = Satellite(settings.remotedb.server)
-        sat.enable_ipv6_http_proxy()
+        sat.enable_satellite_ipv6_http_proxy()
```

Comment:
```suggestion
        sat.enable_satellite_ipv6_http_proxy()
```
Arent we duplicating the `sat` here?

The object name `sat` itself prouds the reference onto which HTTP proxy is being enabled. Hence the explicit `satellite` keyword in function name should now be needed.

Reply:
I don't understand what you mean; your code suggestion has no change.

Reply:
```suggestion
        sat.enable_ipv6_http_proxy()
```

Reply:
Isn't it better to be explicit in the name? This method does setup related to Satellite - it changes Satellite's settings through API. 
I, as a code reader, find it more straightforward to read `enable_satellite_ipv6_http_proxy` or `enable_sat_ipv6_http_proxy` as it tells me that it does something with Satellite itself.

The `enable_ipv6_http_proxy` method can be defined in the `ContentHost` class to do something completely different from modifying Satellite settings.

Host object methods count is constantly increasing, therefore I advocate for explicitness - for keeping `satellite` or `sat` in the method name.

Reply:
i agree that `satellite` is redundant here, the vast majority of the functionality in this repo acts against a Satellite and this method is defined under a Satellite class. 

how about we really clarify what's going on with the name by changing the name to reflect that this is updating proxy settings to use an ipv6 proxy.

perhaps something like `ensure_ipv6_proxy_settings`